README.md

```diff
@@ -1,8 +1,16 @@
-# Getting Started with Create React App
+# Country List
 
-This project was bootstrapped with [Create React App](https://github.com/facebook/create-react-app).
+This project shows country list with react.
 
-## Available Scripts
+## Assumptions
+
+1. Minimal screen width: 375px (iphone SE)
+2. Maximal screen width: 1920px
+3. Clients use the latest browsers (Chrome, Edge)
+4. Google MAP API key is not exposed
+5. Client screen width is fixed when viewing map page
+
+## How to run
```

Comment:
After running `npm i` and then `npm start`, I get:
<img width="1573" alt="image" src="https://user-images.githubusercontent.com/60858938/231019192-be64e2cd-870a-4b76-a536-2d16c6bb7c45.png">

Reply:
Can you please let me know what I am missing?